tests/integration/targets/nd_delta_analysis/tasks/main.yml

```diff
@@ -3,26 +3,30 @@
 
 # GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-- name: Test that we have an Nexus Dashboard Insights Group and Site Name defined
-  fail:
-    msg: "Please define the following variables: ig_name, site_name."
-  when: ig_name is not defined or site_name is not defined
```

Comment:
I would add this back with the proper used variables like insights_group and site_name